mqtt_as/mqtt_as.py

```diff
@@ -132,6 +133,8 @@ def __init__(self, config):
         self._sock = None
         self._sta_if = network.WLAN(network.STA_IF)
         self._sta_if.active(True)
+        if self.hostname:
```

Comment:
Specifying this config option causes an exception on Pyboard D. Options are:
 1. Accept the fact and document it (no RAM burden).
 2. Have `if self.hostname and not PYBOARD:` i.e. silently ignore.
 3. In the case of PYBOARD print a warning and continue (message costs RAM on ESP8266).

What's your view?